.github/release.yml

```diff
@@ -0,0 +1,30 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+      - tests-only
+  categories:
+    - title:  ':tada: Exciting New Features'
+      labels:
+        - '*'
+      exclude:
+        authors:
+          - renovate[bot]
```

Comment:
🤔 Shouldn't we exclude the labels of the other categories here? So e.g. a PR labeled with `bug` is not listed as an exciting new feature.

Reply:
This applies to the other comments too, all of these were put in place to test how their implementation reacts to misconfiguration.